src/components/BeforeGame.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import Input from './Input';
+import Button from './Button';
+
+import boom from '../assets/img/boom.png';
+
+
+class BeforeGame extends Component {
+    constructor(props) {
+        super(props);
+
+        this.onInput1Change = this.onInput1Change.bind(this);
+        this.onInput2Change = this.onInput2Change.bind(this);
+        this.onBeginGame = this.onBeginGame.bind(this);
+
+        this.state = {
+            player1: '',
+            player2: ''
+        }
+    }
+
+    onInput1Change(e) {
+        this.setState({
+            player1: e.target.value
+        });
+    }
+
+    onInput2Change(e) {
+        this.setState({
+            player2: e.target.value
+        });
+    }
+
+    onBeginGame() {
+        this.props.onStartGame(this.state.player1, this.state.player2);
+    }
+
+    render() {
+        console.log(this.state);
```

Comment:
console log shouldn't be merged into master to avoid not nice cleanings in the future